web_widget_digitized_signature/__manifest__.py

```diff
@@ -25,4 +27,6 @@
         'static/src/xml/digital_sign.xml',
     ],
     'installable': True,
+    'development_status': 'Stable',
+    'maintainers': ['cubells'],
```

Comment:
@cubells must accept this

Reply:
@pedrobaeza If he doesn't, what do you suggest? He wrote the module, right?

Reply:
Well, as said, he must accept this charge